frontend/src/main.js

```diff
@@ -30,6 +30,11 @@ loadAllRules(i18n)
 
 addNavigationGuards(router, store, apolloProvider.defaultClient)
```

Comment:
What happens if store is null or undefined here? Or is this not the behavior when blocking everything?